src/main/java/uk/ac/ebi/eva/pipeline/jobs/steps/tasklets/VepAnnotationGeneratorStep.java

```diff
@@ -17,69 +17,59 @@
 
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.springframework.batch.core.JobParameters;
 import org.springframework.batch.core.StepContribution;
 import org.springframework.batch.core.scope.context.ChunkContext;
 import org.springframework.batch.core.step.tasklet.Tasklet;
 import org.springframework.batch.repeat.RepeatStatus;
+import org.springframework.beans.factory.annotation.Autowired;
 
-import uk.ac.ebi.eva.pipeline.parameters.JobParametersNames;
-import uk.ac.ebi.eva.utils.VepUtils;
+import uk.ac.ebi.eva.pipeline.parameters.AnnotationParameter;
 
 import java.io.BufferedInputStream;
 import java.io.FileOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.OutputStream;
 import java.util.Arrays;
-import java.util.Map;
 import java.util.zip.GZIPOutputStream;
 
 /**
  * Tasklet that runs @see <a href="http://www.ensembl.org/info/docs/tools/vep/index.html">VEP</a> over a list of
  * coordinates of variants and nucleotide changes to determines the effect of the mutations.
  * <p>
  * Input: file listing all the coordinates of variants and nucleotide changes like:
- * 20	60343	60343	G/A	+
- * 20	60419	60419	A/G	+
- * 20	60479	60479	C/T	+
+ * 20  60343  60343  G/A    +
+ * 20  60419  60419  A/G    +
+ * 20  60479  60479  C/T    +
  * ...
  * <p>
  * Output: file containing the VEP output
- * 20_60343_G/A	20:60343	A	-	-	-	intergenic_variant	-	-	-	-	-	-
- * 20_60419_A/G	20:60419	G	-	-	-	intergenic_variant	-	-	-	-	-	-
- * 20_60479_C/T	20:60479	T	-	-	-	intergenic_variant	-	-	-	-	-	rs149529999	GMAF=T:0.0018;AFR_MAF=T:0.01;AMR_MAF=T:0.0028
+ * 20_60343_G/A    20:60343   A  -  -  -  intergenic_variant -  -  -  -  -  -
+ * 20_60419_A/G    20:60419   G  -  -  -  intergenic_variant -  -  -  -  -  -
+ * 20_60479_C/T    20:60479   T  -  -  -  intergenic_variant -  -  -  -  -  rs149529999    GMAF=T:0.0018;AFR_MAF=T:0.01;AMR_MAF=T:0.0028
```

Comment:
maybe we should add here "@code" tags to avoid automatic formatting from tabs to spaces, given that the tabs are part of the fileformat specification.